src/App.js

```diff
@@ -32,9 +35,23 @@ export default function ActivityApp() {
   //----- -----
   return (
     <Appcontainer>
-      <DurationFilter onFilterDurationReset={filterDurationReset} onFilterDurationValue={setDurationFilterValue} />
-      <TypeFilter onFilterTypeReset={filterTypeReset} onFilterTypeValue={setTypeFiltervalue} />
-      <ActivityCards activities={filteredAll} onSetActivities={setActivities} />
+      {isHidden === false && (
```

Comment:
You do not need to proof if `isHidden === false`, it is enough to proof if isHidden is falsy, so you can write it like this:

```suggestion
      {isHidden && (
```